.github/workflows/ci.yml

```diff
@@ -13,44 +13,53 @@ on:
 jobs:
   test:
     name: Test
-    runs-on: macOS-latest
+    runs-on: macos-latest
     strategy:
       matrix:
-        platform: ['iOS Simulator,name=iPhone 8']
+        platform: ['iOS Simulator,name=iPhone 14']
     steps:
       - uses: actions/checkout@v2
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: '13.4.1'
```

Comment:
If you target to iPhone 14, Xcode 14.0.1 or 14.1 is needed. https://developer.apple.com/support/xcode/